app/controllers/api_resource_comments_controller.rb

```diff
@@ -4,4 +4,15 @@ def show
     comments = Resource.find(resource_id)&.resource_comments
     render json: comments, only: %i[id content resource_id user_id]
   end
+
+  def create
+    content = params['content']
+    resource = Resource.find(params['resource_id'])
+    comment = ResourceComment.create!(
+      content:,
+      user: current_user,
+      resource_id: resource.id
+    )
+    render json: comment, only: %i[id content resource_id], status: :created
```

Comment:
I think the response should return the user_id because this field it's part of the ResourceComment objects.

Reply:
included as well